### DISCUSSION TITLE
How to set the path for ProDy in VMD

### DISCUSSION THREAD

**Discussion by kiran923** [prody/ProDy#2166] [Archived Issues]
how to set the path for ProDy in VMD
![error](https://github.com/prody/ProDy/assets/57478452/1b7fdfa4-4218-483f-9e1e-fcbd40d22757)

usage: prody.exe [-h] [-c] [-v]
                 {anm,gnm,pca,eda,align,blast,biomol,catdcd,contacts,fetch,select,energy,clustenm}
                 ...
prody.exe: error: argument {anm,gnm,pca,eda,align,blast,biomol,catdcd,contacts,fetch,select,energy,clustenm}: invalid choice: 'C:/Users/DST-SERB/AppData/Local/Programs/Python/Python310/Scripts/prody.exe' (choose from 'anm', 'gnm', 'pca', 'eda', 'align', 'blast', 'biomol', 'catdcd', 'contacts', 'fetch', 'select', 'energy', 'clustenm')
usage: prody.exe [-h] [-c] [-v]
                 {anm,gnm,pca,eda,align,blast,biomol,catdcd,contacts,fetch,select,energy,clustenm}
                 ...
prody.exe: error: argument {anm,gnm,pca,eda,align,blast,biomol,catdcd,contacts,fetch,select,energy,clustenm}: invalid choice: 'C:/Users/DST-SERB/AppData/Local/Programs/Python/Python310/Scripts/prody.exe' (choose from 'anm', 'gnm', 'pca', 'eda', 'align', 'blast', 'biomol', 'catdcd', 'contacts', 'fetch', 'select', 'energy', 'clustenm')
    while executing
"exec $::NMWiz::pybin $::NMWiz::prody anm --quiet -s $selstr {*}$args "$pdbfn""
    (procedure "::NMWiz::prodySubmitANMjob" line 42)
    invoked from within
"::NMWiz::prodySubmitANMjob"
    (procedure "::NMWiz::prodySubmitJob" line 22)
    invoked from within
"::NMWiz::prodySubmitJob"
    invoked from within
".nmwizprody.submitFrame.prodySubmit invoke"
    ("uplevel" body line 1)
    invoked from within
"uplevel #0 [list $w invoke]"
    (procedure "tk::ButtonUp" line 24)
    invoked from within
"tk::ButtonUp .nmwizprody.submitFrame.prodySubmit"
    (command bound to event)

**Comment by jamesmkrieger** [comment]
VMD has clearly found ProDy as it is complaining about the usage instead of asking you to set the path

We can't help you if you don't give us more details about what you were trying to do

**Comment by AnthonyBogetti** [comment]
Hello @kiran923, please provide more details about your workflow with ProDy in VMD so that we can be of assistance. We are happy to help. If your issue is resolved, please let us know as well. Thanks.
